src/YoutubeDl.php

```diff
@@ -116,9 +116,12 @@ public function download(Options $options): VideoCollection
         $process->run(function (string $type, string $buffer) use (&$currentVideo, &$parsedData, &$progressTarget): void {
             ($this->debug)($type, $buffer);
 
-            if (preg_match('/\[(.+)]\s(.+):\sDownloading webpage/', $buffer, $match) === 1) {
+            if (preg_match('/\[(.+)]\s(.+):\sDownloading webpage/', $buffer, $match) === 1
+                || preg_match('/\[(.+)]\s(.+):\sDownloading pc webpage/', $buffer, $match) === 1
+            ) {
                 if ($currentVideo !== null) {
                     $parsedData[] = $currentVideo;
+                    $currentVideo = null;
                 }
 
                 $currentVideo['extractor'] = $match[1];
```

Comment:
```suggestion
                $currentVideo['extractor'] = $match['extractor'];
                $currentVideo['id'] = $match['id'];
```

Reply:
can't suggest for two lines, so just `$currentVideo['id'] = $match[2];` with `$currentVideo['id'] = $match['id'];`